### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by Carlos-Tirado**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,10 +1,136 @@
...
+    font-size: 10px;
+    font-weight: bold;
+    text-align: center;
+    border-radius: 0.625em;
+    position: absolute;
+    left: 73%;
+    bottom: 59%;
+}
+
+button-join {
```
Comment (samrespass) [inline src/App.css:36]: We'll wanna work on consolidating these button styles some to reduce file bloat, tho not an urgent issue

File: src/components/HeaderMenu.js [hunk 2]
```diff
@@ -31,24 +33,21 @@ function Header({ token, setToken }) {
             )
             .then(() => {
                 setToken(null);
-                // localStorage.clear();
-                console.log("logout");
+                localStorage.clear();
```
Comment (racheldk) [inline src/components/HeaderMenu.js:36]: What does localStorage.clear() do? I haven't seen that before.

File: src/components/gamesList.js [hunk 3]
```diff
@@ -168,78 +168,81 @@ export default function GamesList({ token, games, listType, listTitle }) {
...
+                    )}
+
                     </div>
+                    
+                    
+            <button-showmore
+                onClick={() => {
+                    handleOpenModal(game);
+                }}
+            >Show more</button-showmore>
```
Comment (racheldk) [inline src/components/gamesList.js:195]: I haven't seen specific types of buttons like button-showmore and button-join before. Is this like having <button className="button-showmore">  ?

Comment (Carlos-Tirado) [reply src/components/gamesList.js:195]: yeah, in order to give them style I renamed those buttons which are actually working as before .

File: src/App.css (hunk 1 above)
Comment (Carlos-Tirado) [reply src/App.css:36]: OK, you might want to show me about later.

File: src/components/HeaderMenu.js (hunk 2 above)
Comment (Carlos-Tirado) [reply src/components/HeaderMenu.js:36]: I left it there by mistake .It does nothing in the code,

**Pull request merged**